### DISCUSSION TITLE
100

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#11] [General]
# 潮流周刊第100期 - 无忧孩童

在湖南的小路上看到一群小孩在无忧的玩，还有那种巨高兴在路上打滚的，很是快乐，潮流周刊也 100 期了，希望你也快乐～

https://weekly.tw93.fun/posts/100-%E6%97%A0%E5%BF%A7%E5%AD%A9%E7%AB%A5

**Comment by hallestar** [comment]
100期诶，好赞👍

**Reply by tw93** [reply]
哈哈 谢谢喜欢

**Comment by eryajf** [comment]
希望周刊内的外部链接能够在新窗口打开，这样不会影响当前周刊的浏览

**Reply by tw93** [reply]
谢谢反馈 很有必要 晚上安排

**Reply by tw93** [reply]
已经支持外部链接新窗口打开

**Comment by KrabsWong** [comment]
通过订阅 github 仓库 README 文件变更来判断更新的任务，终于可以停了 😂

**Reply by tw93** [reply]
哈哈哈，你喜欢就好，我也是强迫症，之前算是兼容方案了

**Comment by ynymying** [comment]
头图下载下来后能看到完整EXIF信息，不知道这个会不会对网站所有者造成某种意义上的隐私泄露？
